README.md

```diff
@@ -1,99 +1,254 @@
-> 🚨 Deprecation Notice (Effective Q1 2025)
-> We’re migrating all task definitions from the old superchain-ops repo (`tasks/`) into the new improvements repo (`src/improvements/tasks/`).
->
-> • New tasks: please use the [improvements flow](./src/improvements) and refer to the [README](./src/improvements/README.md) for more information.
-> • Need help? Reach out in [#superchain-ops](https://discord.com/channels/1244729134312198194/1342572064175030293) on Discord.
```

Comment:
NIT:I saw that removed the "need help", we don't want to point people on discord anymore?

Reply:
Actually we should remove the discord link, since the server is invite-only. We should instead have people create issues if they need assistance